### DISCUSSION TITLE
does blurred faces in the SAM dataset make faces worse in this model?

### DISCUSSION THREAD

**Discussion by habanerogossip** [PixArt-alpha/PixArt-alpha#16] [Q&A] [answered]
I was looking at the SAM dataset at https://segment-anything.com/dataset/index.html and noticed that all the faces of people are blurred. Does this cause faces to be worse in your model?

**Comment by lawrence-cj** [comment, marked as answer]
Since the SAM dataset is used only for pretraining, It will indeed make the SAM pretrained models generate blured human face. However, there are plenty of normal human face samples in the JourneyDB and Internal Data, so there is no influence on our final model. Besides, we are working on filtering out the blured human face in SAM to solve the problem according to if there is "human" mentioned in the LLaVA-captions.

**Comment by habanerogossip** [comment]
ok sounds good. Ya maybe just add a "blurred face" caption for all the images that apply.
